### PR TITLE
More Doxygen comment improvements.

### PR DESCRIPTION
**Pull request by markdroth**
Some additional Doxygen fixes, as per #11022.

### CODE REVIEW

File: include/grpc++/server_builder.h [hunk 1]
```diff
@@ -71,7 +71,13 @@ class ServerBuilder {
   ServerBuilder();
   ~ServerBuilder();
 
-  enum SyncServerOption { NUM_CQS, MIN_POLLERS, MAX_POLLERS, CQ_TIMEOUT_MSEC };
+  /// Options for synchronous servers.
+  enum SyncServerOption {
+    NUM_CQS,         // Number of completion queues.
```
Comment (KindDragon) [inline include/grpc++/server_builder.h]: Are you sure with doxygen syntax for enum values? https://stackoverflow.com/a/13756348/61505

Comment (markdroth) [inline include/grpc++/server_builder.h]: Thanks for catching this.  Fixed.

File: include/grpc++/server_builder.h [hunk 2]
```diff
@@ -73,10 +73,10 @@ class ServerBuilder {
 
   /// Options for synchronous servers.
   enum SyncServerOption {
-    NUM_CQS,         // Number of completion queues.
-    MIN_POLLERS,     // Minimum number of polling threads.
-    MAX_POLLERS,     // Maximum number of polling threads.
-    CQ_TIMEOUT_MSEC  // Completion queue timeout in milliseconds.
+    NUM_CQS,         /// Number of completion queues.
```
Comment (KindDragon) [inline include/grpc++/server_builder.h]: You forgot less sign `///<` (https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html check "Putting documentation after members")

Comment (markdroth) [inline include/grpc++/server_builder.h]: Fixed. Thanks.

**Pull request merged**